### PR TITLE
more instructions on how to enable Dependabot

### PR DESCRIPTION
**Pull request by jakecoffman**
[rendered](https://github.com/dependabot/demo/blob/2713761519694a4230802a74561264dc7fb2c853/README.md)

It's not clear what you need to do after forking so I've added instructions on how to enable both security and version updates.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,39 @@
...
+- Dependabot will now start creating PRs for detected security vulnerabilities
+- Go into the **Security** tab and click **Dependabot** in the left hand side navigation to see what Dependabot is working on
+
+<img width="929" alt="screenshot showing Dependabot working on Security Updates" src="https://github.com/dependabot/demo/assets/886768/9295c61a-631b-4c56-9c00-ff078874f362">
+
+After about 5 minutes you should see some PRs open. Merge them and the Securty Alerts will close 🎉
+
+### Enabling Version Updates
+
+This demo includes a `dependabot.yml` which configures [Version Updates](https://docs.github.com/github/administering-a-repository/keeping-your-dependencies-updated-automatically), but forks don't automatically start with Dependabot enabled.
```
Comment (jeffwidman) [inline README.md:28]:
```suggestion
This demo includes a `dependabot.yml` which configures [Version Updates](https://docs.github.com/github/administering-a-repository/keeping-your-dependencies-updated-automatically), but [forks don't automatically start with Dependabot enabled](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuring-dependabot-version-updates#enabling-version-updates-on-forks).
```

File: README.md [hunk 2]
```diff
@@ -1,12 +1,39 @@
...
     </picture>
-</h1>
+</div>
 
-# Dependabot Demo Repository
+## Dependabot Demo Repository
 
 This repo contains some projects with outdated dependencies. Fork it to try out
 Dependabot :dependabot:!
+
```
Comment (jeffwidman) [inline README.md:13]:
```suggestion

Note:  [Forks default to Dependabot disabled](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuring-dependabot-version-updates#enabling-version-updates-on-forks), so a couple of additional steps are required.

```

**Review by jeffwidman — APPROVED** [review]

**Comment by jeffwidman** [comment]
@jakecoffman didn't realize you had auto-merge on... take a look at my feedback above and see whether you want to incorporate it or not.

I just thought it'd be good to hyperlink additional details to reduce confusion for those who don't understand why forks are disabled, or whether they'd need to do these steps for non-forked repos.

**Comment by jakecoffman** [comment]
I linked to the parent page in the same sentence, it's probably good enough. Someone curious will find it if they want to know.

**Review by masumbillah30 — APPROVED** [review]

**Pull request merged**